foxpuppet/windows/browser/tab.py

```diff
@@ -0,0 +1,8 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+class Tab(object):
+    def __init__(self, selenium, *args, **kwargs):
+        self.selenium = selenium
```

Comment:
Also, this init has nothing custom, so it can be omitted. If there are likely to be a lot of regions within the tab bar then we might also want to consider making tab_bar a module and moving tab.py into it.